tutorials/workzone-enrich-5-integrate-card/workzone-enrich-5-integrate-card.md

```diff
@@ -39,13 +39,13 @@ You've already deployed a UI5 integration card. Now in this tutorial, you'll ena
 
     >Note: if you don't see your deployed card, you may need to refresh the browser.
```

Comment:
Please don't use the label "note" for a note, because the formatting already makes it a note. If you want to emphasize the note, use labels like "important".